scripts/pre_commit_linter.py

```diff
@@ -0,0 +1,155 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Pre-commit script for Oppia.
+
+This script uses the JSCS node module to lint JavaScript code
+It returns and autofix errors
+
+IMPORTANT NOTES:
+
+1.  Before running this script, you must install third-party dependencies by
+    running
+
+        bash scripts/start.sh
+
+    at least once.
+
+2.  This script should be run from the oppia root folder:
+
+        python scripts/pre_commit_linter.py
+
+ Note that the root folder MUST be named 'oppia'.
+ """
+
+__author__ = "Barnabas Makonda(barnabasmakonda@gmail.com)"
+
+import argparse
+import os
+import subprocess
+import sys
+import time
+
+
+_PARSER = argparse.ArgumentParser()
+_PARSER.add_argument(
```

Comment:
We don't currently use this, though, right? If not, it's fine to leave it out for now; we can add the option later in a separate pass.